tests/processes/test_convert.py

```diff
@@ -1030,6 +1030,9 @@ def test_convert_input_values_schema_from_old():
         {"id": "test10", "value": "more"},
         {"id": "test11", "href": "/data/file1.txt", "format": {"mediaType": "text/plain"}},
         {"id": "test11", "href": "/data/file2.txt", "format": {"mediaType": "text/plain"}},
+        {"id": "test12", "href": "/data/file1.txt", "format": {"mediaType": "text/plain"}},
+        {"id": "test12", "href": "/data/file2.txt", "format": {"mediaType": "text/plain"}},
+        {"id": "test12", "href": "/data/file3.txt", "format": {"mediaType": "text/plain"}},
```

Comment:
A few more test cases should be added to make sure.

13. \>2 literal values
14. \>2 literal values with >2 repeated IDs
15. same as 14, but with complex href

Reply:
Maybe it's just the terminology, but I'm not sure I follow what the difference is between 13 and 14? Aren't the IDs already repeated? (id: test12, etc.)

For 15, complex href would be something like directory type?

Reply:
You can have `[{"id": "testX", "value": "x"}, {"id": "testX", "value": ["x", "x"]}]`. 
The whole ID/value object repeated, or multiple values within a given object.
They were already tested, but with only 2 values instead of >2.

And for 15, I don't even remember if the code can handle
`{"id": "x", "href": [ "/path1", "/path2"], "format": "fmt"}`
(ok if it doesn't)

Good idea to test directories as well.